cgi-bin/DW/Controller/Support/Faq.pm

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Support::Faq
+#
+# This controller is for the Support FAQ page.
+#
+# Authors:
+#      hotlevel4 <hotlevel4@hotmail.com>
+#
+# Copyright (c) 2015 by Dreamwidth Studios, LLC.
+#
+# This is based on code originally implemented on LiveJournal.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+
+package DW::Controller::Support::Faq;
+
+use strict;
+use warnings;
+use vars qw(%GET);
+
+use DW::Controller;
+use DW::Routing;
+use DW::Template;
+
+DW::Routing->register_string( '/support/faq', \&faq_handler, app => 1 );
+
+sub faq_handler {
+    my $r = DW::Request->get;
+
+    my ( $ok, $rv ) = controller( anonymous => 1, form_auth => 1 );
+    return $rv unless $ok;
+
+    my $remote = LJ::get_remote();
+    my $user;
+    my $user_url;
+
+    my $vars = {};
+    
+    my $dbr = LJ::get_db_reader(); 
+    my $sth;
+    my %faqcat; 
+    my %faqq;
+    my $ret = "";    
+    
+    $sth = $dbr->prepare("SELECT faqcat, faqcatname, catorder FROM faqcat ".
```

Comment:
Note: not going to go over all instances (and I did notice that there are some with the added whitespace already!) but please take another look to clean up the remaining :)